templates/email/data-breach-notification.php

```diff
@@ -2,27 +2,27 @@
 echo sprintf(
 	/* translators: 1: Email content, 2: Nature of data breach, 3: Contact details for data protection officer, 4: Likely consequences of breach, 5: Measures taken */
 	esc_html__(
-'%s
+		'%1$s
```

Comment:
If you add that space, it is going to show on the actual email. Review the email templates for this.

Reply:
Not, because I puts the space before the single quote.